package.json

```diff
@@ -27,6 +28,7 @@
     "!(*.{js,ts})": "prettier --write --ignore-unknown"
   },
   "devDependencies": {
+    "@types/mocha": "^10.0.7",
```

Comment:
[Explanation] This isn't strictly necessary with `"noImplicitAny": false`. But it's just good to have types for dependencies in general.